client/blocks/edit-gravatar/index.jsx

```diff
@@ -204,7 +208,7 @@ export class EditGravatar extends Component {
 		const icon = ( user.email_verified ? 'cloud-upload' : 'notice' );
 		const buttonText = ( user.email_verified
 			? translate( 'Click to change photo' )
-			: translate( 'Verify email first' )
+			: translate( 'Verify your email' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Verify your email address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/edit-gravatar-on-desktop)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).